net/bridge.go

```diff
@@ -514,6 +518,42 @@ func configureIPTables(config *BridgeConfig) error {
 	return nil
 }
 
+type NoMasqLocalTracker struct {
+	ips   ipset.Interface
+	owner types.UID
+}
+
+func NewNoMasqLocalTracker() *NoMasqLocalTracker {
+	return &NoMasqLocalTracker{
+		// TODO(mp) Reuse ipset object as each time calling New creates a test set.
+		ips:   ipset.New(common.LogLogger()),
+		owner: types.UID(0), // dummy ipset owner
+	}
+}
+
+func (t *NoMasqLocalTracker) String() string {
+	return "no-masq-local"
+}
+
+func (t *NoMasqLocalTracker) HandleUpdate(prevRanges, currRanges []address.Range, local bool) error {
```

Comment:
I think this should check that `local` is true

Reply:
Good catch!